static/app/components/onboarding/frameworkSuggestionModal.tsx

```diff
@@ -387,6 +398,17 @@ const Actions = styled('div')`
 `;
 
 export const modalCss = css`
+  [role='document'] {
+    display: flex;
+    flex-direction: column;
+    max-height: 80vh;
+    min-height: 500px;
+  }
+  section {
+    display: flex;
+    flex-direction: column;
+    min-height: 0;
+  }
```

Comment:
I would have preferred not to overwrite the modal styles like this, but I did not find any other solution with the current structure 🤔

Reply:
Can we please add a comment explaining why we are doing this?